MATLAB/Demos/FDK_half.m

```diff
@@ -0,0 +1,192 @@
+function [res]=FDK_half(proj,geo,angles,varargin)
+%FDK for displaced detector
+%
+%--------------------------------------------------------------------------
+%--------------------------------------------------------------------------
+% This file is part of the TIGRE Toolbox
+% 
+% Copyright (c) 2015, University of Bath and 
+%                     CERN-European Organization for Nuclear Research
+%                     All rights reserved.
+%
+% License:            Open Source under BSD. 
+%                     See the full license at
+%                     https://github.com/CERN/TIGRE/blob/master/LICENSE
+%                     and
+%                     https://www.mathworks.com/matlabcentral/fileexchange/view_license?file_info_id=35548
+%
+% Contact:            tigre.toolbox@gmail.com
+% Codes:              https://github.com/CERN/TIGRE/
+% Coded by:           (Original FDK) Kyungsang Kim, modified by Ander Biguri 
+%                     (FDK_half) Yi Du (yi.du@hotmail.com)
+%--------------------------------------------------------------------------
+%% Parameter pasring
+[filter,parker]=parse_inputs(proj,geo,angles,varargin);
+geo=checkGeo(geo,angles);
+geo.filter=filter;
+
+%Input is data,geosize,angles
+if size(geo.offDetector,2)==1
+    offset=repmat(geo.offDetector,[1 size(angles,2)]);
+else
+    offset=geo.offDetector;
+end
+
+%% Zero-padding to avoid FFT-induced alising
+[zproj, zgeo, theta] = zeropadding(proj, geo);
+
+%% Preweighting using Wang function
+[proj_w, pre_w] = preweighting(zproj, zgeo, theta);
+
+%% Replace original proj and geo
+proj = proj_w;

```

Comment:
We should replace them in line 39 to save memory. Perhaps `zproj` is not needed either, replacing is safe.